app/controllers/AnnotationIOController.scala

```diff
@@ -72,34 +72,25 @@ class AnnotationIOController @Inject()(val messagesApi: MessagesApi)
     val parseSuccess = parseResultsPrefixed.filter(_.succeeded)
 
     if (!parsedFiles.isEmpty) {
-      val tracings = parseSuccess.flatMap(_.tracing)
+      val tracings = parseSuccess.flatMap(_.bothTracingOpts)
       val (skeletonTracings, volumeTracings) = NmlService.splitVolumeAndSkeletonTracings(tracings)
```

Comment:
can you rename the `volumeTracing` to make it a bit more clear that it also stores a string.